products/datavirt/get-started.adoc

```diff
@@ -1,14 +1,6 @@
 :awestruct-layout: product-get-started
 
-## Extra installation instructions
+## Extra section
 
-[start=3]
-. For JBoss Data Virtualization 6.1 Beta, open a terminal window and navigate to where the GUI installer was downloaded.  Then enter the command in to launch the GUI installer:
+For Installation Instructions, please refer to the instructions in the JBoss Data Virtualization 6.1 Installation Guide: https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_Data_Virtualization/
```

Comment:
You need to text for the link, rather than having the url appear on the page. Take a look at other links in this file for examples.

As we need this merging asap, I've created: https://github.com/jboss-developer/www.jboss.org/pull/833